hack/lib.sh

```diff
@@ -37,7 +37,7 @@ KUBEBUILDER_PATH=$OUTPUT_BIN/kubebuilder
 KUBEBUILDER_BIN=$KUBEBUILDER_PATH/bin/kubebuilder
 KUBEBUILDER_VERSION=2.3.2
 KUSTOMIZE_BIN=$OUTPUT_BIN/kustomize
-KUSTOMIZE_VERSION=3.5.4
```

Comment:
It seems this line of change is not related to this issue, is it required?

Reply:
This is not required by this issue. When I run the e2e test on my local environment, require `KUSTOMIZE` more hight version